protobuf/endpoints/execute_function.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+import "models/registered_function.proto";
+import "models/status.proto";
+import "models/value.proto";
+
+message ExecuteFunctionRequest {
+  optional RegisteredFunction function_to_execute = 1;
+  repeated Value function_parameters = 2;
```

Comment:
if it's potentially important that a client can tell which params led to which output, it might be nice to use [maps](https://protobuf.dev/programming-guides/proto3/#maps) for something like `map<string, Value> function_parameters = 2;`